mantidproject/mantid
# Inject the correct PYTHONPATH into error reporter

This way uses the exact same mechanism that is used for launching mantid so it should be updated as necessary already. Only the linux launching scripts were modified.

**To test:**

In order to actually see the error reporter come up, you will need to set `usagereports.enabled=1` in your `Mantid.user.properties`. Then startup `mantidplot` or `mantidworkbench` and run
```python
Segfault(Dryrun=False)
```

Fixes #24762

*This does not require release notes* because it is a minor bug in fixing the error reporter.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
